.gitlab-ci-check-docker-build.yml

```diff
@@ -209,7 +209,7 @@ publish:image-multiplatform:
   rules:
     - if: $MULTIPLATFORM_BUILD != "true"
       when: never
-    - if: '$CI_COMMIT_BRANCH =~ /^(main|master|staging|production|feature-.+)$/'
+    - if: '$CI_COMMIT_BRANCH =~ /^(main|master|staging|production|feature-.+|v?\\d+\\.\\d+\\.x)$/'
```

Comment:
@oldgiova Thinking about any project that may be using this template, do you see any issue in this? 🤔